### PR TITLE
Add @znewman01 to reusable-release

### PR DESCRIPTION
**Pull request by znewman01**
I am a Cosign owner and occasionally need to cut a Cosign release.

Signed-off-by: Zachary Newman <zjn@chainguard.dev>

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]

**Review by haydentherapper — APPROVED** [review]
We should probably move these reusable workflows to a dedicated project at some point. 

Also Lgtm, but we need to rethink who should be able to cut releases, since this isn’t sustainable. We have an issue in public good instance if anyone wants to chime in!

File: .github/workflows/reusable-release.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
       RELEASE_TAG: ${{ inputs.release_tag }}
     steps:
       - name: Check actor access
-        if: ${{ !contains( fromJson('["bobcallaway","cpanato","dlorenc","lukehinds","priyawadhwa","vaikas","haydentherapper","afeddersen"]'), github.actor ) }}
+        if: ${{ !contains( fromJson('["bobcallaway","cpanato","dlorenc","lukehinds","priyawadhwa","vaikas","haydentherapper","afeddersen","znewman01"]'), github.actor ) }}
```
Comment (haydentherapper) [inline .github/workflows/reusable-release.yml:45]: Could we remove afeddersen?

**Comment by cpanato** [comment]
working to remove this and use github actions only and not more cloudbuild, will try to get this in the xmas time ready

**Review by priyawadhwa — APPROVED** [review]

**Review by priyawadhwa — APPROVED** [review]

**Comment by znewman01** [comment]
Addressed comments so ready for re-approval.

+1 to the comments about sustainability here, thanks @cpanato you're doing incredible work as usual 🎉

**Pull request merged**